packages/web-components/src/player-component/UI/ui.class.ts

```diff
@@ -201,6 +201,63 @@ export class OverflowMenu extends shaka.ui.OverflowMenu {
     }
 }
 
+export class MetaDataButton extends shaka.ui.OverflowMenu {
+    private svg: SVGSVGElement;
+    private path: SVGPathElement;
+    private isBoxOn: boolean;
+    private isAttributesOn: boolean;
+
+    public constructor(parent: any, 
+                       controls: any, 
+                       private showBoxCallBack: (isOn: boolean) => void, 
+                       private showAttributesCallBack: (isOn: boolean) => void) {
+        super(parent, controls);
+        this.init();
+    }
+
+    public createChildren_() {
+        const showBoundingBox = document.createElement('div');
+        showBoundingBox.innerText = ' Bounding Box ';
+        const BoxCheckbox = document.createElement('input');
```

Comment:
Why are we not using Fast controls here?

Reply:
We are now using FAST checkbox.